README.md

```diff
@@ -59,22 +59,43 @@ snap install microk8s --classic
 MicroK8s includes a `microk8s.kubectl` command:
 
 ```
-microk8s.kubectl get nodes
-microk8s.kubectl get services
+sudo microk8s.kubectl get nodes
+sudo microk8s.kubectl get services
 ```
 
 To use MicroK8s with your existing kubectl:
 
 ```
-microk8s.kubectl config view --raw > $HOME/.kube/config
+sudo microk8s.kubectl config view --raw > $HOME/.kube/config
 ```
 
+#### User access without sudo
+If the *microk8s* user group exist at MicroK8s `snap install` time, users on that group
+will be granted access to `microk8s` commands.
+
+At any point you grant access to a specific group with:
+```
+sudo snap set microk8s user.group=<GROUP_NAME>
+```
+
+For example, to create the `microk8s` group and add a user to it:
+```
+# Create the group
+sudo addgroup microk8s
+# Add a user to the group. The new group will be available on the user's next login.
```

Comment:
Note that `newgrp` and `sg` can be used immediately to permanently change the ruid/euid.